pkg/proc/native/threads_single_step_linux.go

```diff
@@ -0,0 +1,32 @@
+// +build !arm
```

Comment:
This file shoud be called threads_hardware_singlestep_linux.go

Reply:
Got it.